PNA/agilentN5242A.py

```diff
@@ -285,8 +288,27 @@ def freq_sweep(self, c, log=False):
         # wait for sweep to finish
         sparams = yield self.getSweepData(dev, c['meas'])
         returnValue((freq*units.Hz, sparams))
-        
-        
+
+    @setting(102, returns='*v[Hz]*c')
+    def get_trace(self, c, trace='S21'):
+        """Get whatever trace is on the screen.
```

Comment:
This seems weird.  The docstring says "get whatever trace is on the screen", but you have to specify an S parameter as an argument.  What happens if S21 is not on the screen?  Why do I have to know what is on the screen to grab it?